library/common/glfw/text_input_plugin.h

```diff
@@ -19,27 +19,22 @@
 
 #include "library/common/glfw/keyboard_hook_handler.h"
 #include "library/common/internal/text_input_model.h"
-#include "library/include/flutter_desktop_embedding/json_plugin.h"
+#include "library/include/flutter_desktop_embedding/method_channel.h"
+#include "library/include/flutter_desktop_embedding/plugin_registrar.h"
 
 namespace flutter_desktop_embedding {
 
 // Implements a text input plugin.
 //
 // Specifically handles window events within GLFW.
-class TextInputPlugin : public KeyboardHookHandler, public JsonPlugin {
+class TextInputPlugin : public KeyboardHookHandler {
  public:
-  TextInputPlugin();
+  TextInputPlugin(PluginRegistrar *registrar);
   virtual ~TextInputPlugin();
 
-  // Plugin.
-  void HandleJsonMethodCall(const JsonMethodCall &method_call,
-                            std::unique_ptr<MethodResult> result) override;
-
-  // KeyboardHookHandler.
+  // KeyboardHookHandler:
```

Comment:
Should've added this to my previous comment. I asked to remove the extra line since there's no line here. Maybe add one here? :)

Reply:
This is a style I picked up from Chromium (IIRC): all the overridden function declarations from one superclass are declared as a blob, without blank lines or comments (since the declaration comment is at the original declaration). Since it's one blob, the comment is attached to it without a blank line.

In the implementation file, on the other hand, there are always blank lines between function definitions, so the comment needs to be separated so it doesn't appear to be an implementation comment for one function. Maybe doing the implementation header is confusing in C++, but I find it helpful in navigating ObjC files (where it's very common, using #pragma mark) so I thought I'd give it a try here.

Reply:
Makes sense, thanks!